test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
 			}
+			wg.Wait()
```

Comment:
I certainly don't want to make the problem worse, but what would cause 1/20(same proxy) or 1/34(different proxies) to take longer than 30 seconds?

Is the goal of this test really to see if we can handle 5 QPS? That needs to be in the title of the test or otherwise described somewhere.